metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java

```diff
@@ -0,0 +1,50 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+import eu.europeana.cloud.service.dps.DpsTask;
+import java.util.List;
+
+/**
+ * Depublish Plugin.
+ *
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2020-06-16
+ */
+public class DepublishPlugin extends AbstractExecutablePlugin<DepublishPluginMetadata> {
```

Comment:
Knowing that there will be a depublish plugin for records later, perhaps rename this DepublishDatasetPlugin? Or would you prefer to merge these two plugins?

Reply:
In fact, this goes for all enum members that you declared as well (e.g. those called DEPUBLISH would be called DEPUBLISH_DATASET).

Reply:
Actually I was thinking indeed to combine the two in one plugin indeed and we'll have two separate parameters. What do you think?